horizons/gui/tabs/boatbuildertabs.py

```diff
@@ -163,14 +164,10 @@ def update_queue(self, container_active):
 
 			try:
 				icon = Icon(name=icon_name, image=image, helptext=helptext)
-			except RuntimeError,e:
+			except fife.NotFound,e:
```

Comment:
I'd really like this to change to `fife.NotFound as e` or *at least* add a space between the ',' and the 'e'.